zhaquirks/xiaomi/__init__.py

```diff
@@ -625,6 +642,19 @@ def _update_attribute(self, attrid, value):
         super()._update_attribute(attrid, value)
 
 
+class LocalIlluminanceMeasurementCluster(
+    LocalDataCluster, IlluminanceMeasurementCluster
+):
+    """Illuminance measurement cluster based on LocalDataCluster."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
+        if self.AttributeDefs.measured_value.id not in self._attr_cache:
+            # put a default value so the sensor is created
+            self._update_attribute(self.AttributeDefs.measured_value.id, 0)
```

Comment:
This was copied from the `motion_ac02` (Aqara P1 motion sensor) quirk.
A future PR will change that P1 sensor to use this "main" class.
(similar to the above `XiaomiPowerConfigurationPercent` class)